### ISSUE TITLE
NVDA

### ISSUE THREAD

**Issue by BEdenhofer6** [nvaccess/nvda#16899]
I would love, to have a remote feature, built into future releases of the prodject, so us individuals, could remote into each others PC's, instead of downloading, and updating, the remote AddOn all the time. I think, as an NVDA user, it would be wonderful.

**Comment by josephsl** [comment]
Hello,

Thank you for contacting NVDA developers. Our office is currently "closed". Please try again later.

Real answer: please fill out the template to the fullest, and raise issues only after try looking for open issues (or some closed issues) on GitHub first.
Thanks.

**Comment by gerald-hartig** [comment]
Thank you for your submission. Please have a look at the advice on submitting a ticket to the NVDA project in the issue template. Links to more information are available in the template. This will allow us to triage the feature request or issue.

**Comment by Adriani90** [comment]
Closing as duplicate of #3564 and #4390.
